media-sound/ario/ario-1.6.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg-utils
+
+DESCRIPTION="GTK client for MPD inspired by Rhythmbox but much lighter and faster"
+HOMEPAGE="http://ario-player.sourceforge.net"
+SRC_URI="mirror://sourceforge/${PN}-player/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="dbus debug +idle nls taglib zeroconf"
+
+RDEPEND="dev-libs/glib:2
+	dev-libs/libgcrypt:0
```

Comment:
If this gets linked during compile, it needs to be binded with `:=`.

Reply:
on closer inspection it turned out ario 1.6 no longer depends on libgcrypt